xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -324,6 +324,9 @@ void Win32DllLoader::OverrideImports(const std::string &dll)
 
 bool Win32DllLoader::NeedsHooking(const char *dllName)
 {
+  if (!StringUtils::EndsWithNoCase(dllName, "libdvdread.dll"))
```

Comment:
also, check the history of this file: it's been used for PVR etc. We need to be sure that ONLY libdvd is still using this.
https://github.com/mapfau/xbmc/commits/whitelist_dllloader/xbmc/cores/DllLoader/Win32DllLoader.cpp